.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,7 +32,13 @@
 
 - [ ] New files have MPL-2.0 license header.
 - [ ] Request a security/privacy review as needed.
-- [ ] Adequate test coverage exists to prevent regressions 
+- [ ] Adequate test coverage exists to prevent regressions
+- [ ] Build-flag guards are used for code that touches:
+    - Ads
```

Comment:
also Wallet, Referrals, Greaselion and Translate. Maybe also a checkbox to update this list when new buildflags are added?

Reply:
I'd suggest just don't try to be exhaustive here. This already misses a lot of them like crypto wallets, binance, gemini, ipfs, etc.